extension/src/popup/Router.tsx

```diff
@@ -363,9 +361,6 @@ const Outlet = () => {
           <PublicKeyRoute path={ROUTES.mnemonicPhraseConfirmed}>
             <FullscreenSuccessMessage />
           </PublicKeyRoute>
-          <PublicKeyRoute path={ROUTES.pinExtension}>
-            <PinExtension />
-          </PublicKeyRoute>
```

Comment:
we used to have some logic to redirect users to a specific "You're all set" page on Chrome and a different one on Firefox. We now just have one unified view for both, so this `PinExtension` is now no longer needed